src/apps/common/submodules/appSelector/appSelector.scss

```diff
@@ -206,6 +206,10 @@
 
 						&.selected {
 							background-color: $french-pass;
+
+							&:hover {
+								background-color: darken($french-pass, 5%);
```

Comment:
Mmmm, actually I don't think the color is different when hovering a selected item.

Reply:
Yes, it was not in the mock-up either. I'll remove it. It was just that it felt a little awkward to have a color change when hovering the other items, but not when hovering the selected ones.